csharp/src/Drivers/Apache/Spark/SparkHttpConnection.cs

```diff
@@ -164,7 +164,8 @@ protected override Task<TTransport> CreateTransportAsync()
             TConfiguration config = new();
             ThriftHttpTransport transport = new(httpClient, config)
             {
-                ConnectTimeout = HttpRequestTimeout,
+                // TODO: Remove support for HttpRequestTimeoutMilliseconds
+                //ConnectTimeout = HttpRequestTimeoutMilliseconds,
```

Comment:
If `ThriftHttpTransport.ConnectTimeout` is not set, it defaults to 30 seconds. Ideally, this should be the max of Connect/QueryTimeout. However, the `QueryTimeout` can be updated after this setting is initially made. Luckily, it is possible to update the value dynamically : `Connection.Client.OutputProtocol.Transport as ThriftHttpTransport`

Reply:
I pushed changes to address the max of Connect/Query timeout. I don't think we need to handle the dynamically changed value now, do we?

Reply:
That should be safe, for now. I'm sure many applications actaully update the query timeout dynamically.